flake.nix

```diff
@@ -114,4 +126,10 @@
           };
         };
       };
+
+
+  nixConfig = {
```

Comment:
Same here, should be both caches

Reply:
Are you referring to extending the `substituters` and `trusted-public-keys` list here?

https://github.com/holochain/wind-tunnel/blob/main/flake.nix#L133

Reply:
Yes, although if you don't override the whole list, you don't need to add the NixOS cache. I'll probably change that again on Wind Tunnel. Just the Holochain CI

Yes it'd be the same values here, it's just the URL and a public key